src/patterns/components/inputs/datepicker.hbs

```diff
@@ -3,7 +3,7 @@ name: Date Picker
 description:
   The date input is used for collecting date information. This component includes a date picker.
 information:
-  - Spark recommends that this component be used when you expect the entered date to be in the past.
+  - Spark recommends that this component be used when you expect the entered date to be within ~+/- 1 year.
```

Comment:
@motionsuggests sorry to make a request after this is approved, but can you change it to "...the entered date to be roughly one year before or after the current date."

I just want to be explicit and not potentially confuse people because they don't understand the shorthand symbols or something.

Reply:
@yodasw16 Done!